spanner/ddl/ast.go

```diff
@@ -181,17 +252,13 @@ func (ct CreateTable) PrintCreateTable(config Config) string {
 	var col []string
 	var colComment []string
 	var keys []string
-	for i, cn := range ct.ColNames {
+	for _, cn := range ct.ColNames {
 		s, c := ct.ColDefs[cn].PrintColumnDef(config)
-		s = "\n    " + s
-		if i < len(ct.ColNames)-1 {
-			s += ","
-		} else {
-			s += " "
-		}
+		s = "\t" + s + ",\n"
```

Comment:
is this because a trailing , is supported by spanner ?

Reply:
Yes, trailing , is supported by spanner and this change makes the code uniform across both dialects.